Sources/ApolloCodegenLib/FileGenerators/SchemaModuleFileGenerator.swift

```diff
@@ -10,43 +10,24 @@ struct SchemaModuleFileGenerator {
     _ config: ApolloCodegenConfiguration.SchemaTypesFileOutput,
     fileManager: FileManager = FileManager.default
   ) throws {
+
```

Comment:
Commits to this file should not have been included in this PR - ignore it. The next PR, based off this one, addresses file generators.